Source/Charts/Renderers/BarChartRenderer.swift

```diff
@@ -445,12 +445,9 @@ open class BarChartRenderer: BarLineScatterCandleBubbleRenderer
                             px += iconsOffset.x
                             py += iconsOffset.y
                             
-                            ChartUtils.drawImage(
-                                context: context,
-                                image: icon,
-                                x: px,
-                                y: py,
-                                size: icon.size)
+                            context.drawImage(icon,
+                                              atCenter: CGPoint(x: px, y: py),
```

Comment:
will it be possible it's not center? just (x, y)

Reply:
If you look at the method implementation, it uses a centre point. The old parameter naming was misleading.